src/main/java/emu/grasscutter/game/Account.java

```diff
@@ -74,7 +74,11 @@ public void setPlayerId(int playerId) {
 	}
 	
 	public String getEmail() {
-		return email;
+		if(email != null && !email.isEmpty()) {
```

Comment:
Just a side note: email.isEmpty() is the same as the empty string you are returning, so you dont need it in this if statement.